gpg-interface.h

```diff
@@ -3,6 +3,7 @@
 
 #define GPG_VERIFY_VERBOSE	1
 #define GPG_VERIFY_RAW		2
+#define GPG_VERIFY_QUIET	4
 
```

Comment:
Junio wanted us to get rid of this flag. But I think it makes sense to keep it and use it to prevent `print_verify_buffer` from being called.